client/src/featureform/sqlite_metadata.py

```diff
@@ -686,7 +687,7 @@ def get_source_file_last_updated(self, resource_type, name, variant, file_path):
         return res[0] if res else None
 
     def insert_source(self, tablename, *args):
-        stmt = f"INSERT OR IGNORE INTO {tablename} VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)"
+        stmt = f"INSERT OR IGNORE INTO {tablename} VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)"
```

Comment:
Probably another data point for using an ORM for SQLite, no? 🙄

Reply:
yeahhh 100%